src/main/java/seedu/duke/project/Project.java

```diff
@@ -0,0 +1,59 @@
+package seedu.duke.project;
+
+import java.time.LocalDate;
+
+public class Project {
+
+    public static ProjectBacklog backlog;
```

Comment:
What's the reason for using static members? This will be a problem when it comes to multi-project management.

Reply:
Agreed, we can remove the static keyword to allow each instances of the class to have their own variables. Just to better prepare for our v2.

Reply:
Agreed.
Okay so just public variable will be used here? Then won't we have to pass the project object in all the functions?

Reply:
Yeah. Since it is not static, you still need to pass the project object around to be manipulated, and that should be the way since there will be multi-project management in v2. This is because we need to determine which exact project we are manipulating.

Reply:
Just to clarify:
So once project is created and thrown, the command returns the proj variable which is stored in the parser to edit?